src/sql/workbench/services/accountManagement/browser/accountListRenderer.ts

```diff
@@ -129,7 +129,13 @@ export class AccountListRenderer extends AccountPickerListRenderer {
 			// templateData.actions.push(new ApplyFilterAction(ApplyFilterAction.ID, ApplyFilterAction.LABEL), actionOptions);
 		}
 
-		const removeAction = this._instantiationService.createInstance(RemoveAccountAction, account);
-		templateData.actions.push(removeAction, actionOptions);
+		if (account.key.providerId === 'github') {
```

Comment:
That comes from the accountType property on the `az.Account` type. The current options are "work_school" and "personal". I'll add an option for github, but I can't infer the account type from the session since it doesn't specify if the session is for a corporate or personal subscription of Copilot.

I was previously defaulting to "work_school", but Charles had pointed out how that didn't make sense and I agree with him.

Reply:
This is what I was suggesting to update the code to use a default icon if the accountType isn't recognized. And then you can add something for Cheena's suggestion which will use the Github icon when the account is a Github account.

Reply:
Added github icon for github accounts: 
![image](https://github.com/microsoft/azuredatastudio/assets/87730006/db08dbf6-7f70-4295-b8ff-1131ccf72e7a)